components/prize_pool/wikis/prize_pool_legacy_custom.lua

```diff
@@ -0,0 +1,39 @@
+---
+-- @Liquipedia
+-- wiki=mobilelegends
+-- page=Module:PrizePool/Legacy/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local PrizePoolLegacy = Lua.import('Module:PrizePool/Legacy', {requireDevIfEnabled = true})
+
+local CustomLegacyPrizePool = {}
+
+-- Template entry point
+function CustomLegacyPrizePool.run()
+	return PrizePoolLegacy.run(CustomLegacyPrizePool)
+end
+
+function CustomLegacyPrizePool.customOpponent(opponentData, CACHED_DATA, slot, opponentIndex)
+	-- ML has use case of same placement but has different earnings
+
+	if slot['usdprize' .. opponentIndex] then
+		opponentData.usdprize = slot['usdprize' .. opponentIndex]
+	end
+
+	if slot['points' .. opponentIndex] then
+		local param = CACHED_DATA.inputToId['points']
+		CustomLegacyPrizePool._setOpponentReward(opponentData, param, slot['points' .. opponentIndex])
+	end
+
```

Comment:
Looking at the old code, this was only done for localprize & usdprize, not for points